examples/on-conflict-clause/on_conflict_clause.bal

```diff
@@ -1,38 +1,61 @@
 import ballerina/io;
 
-type Student record {|
+type Employee record {|
     readonly int id;
     string name;
+    string role;
     int score;
 |};
 
 public function main() {
-    error onConflictError = error("Key Conflict", message = "record with same key exists.");
-
-    table<Student> students = table [
-            {id: 1, name: "John", score: 100},
-            {id: 2, name: "Jane", score: 150},
-            {id: 1, name: "John", score: 200}
+    table<Employee> employees = table [
+            {id: 1, name: "Mike", role: "Admin", score: 100},
+            {id: 2, name: "John", role: "HR", score: 150},
+            {id: 3, name: "Amy", role: "HR", score: 150},
+            {id: 1, name: "Mike", role: "Admin", score: 200}
         ];
 
-    // The result of the following will be an error since the key `John` is duplicated.
-    map<int>|error studentScores = map from var {name, score} in students
+    // The result of the following will be an error since the key `Mike` is duplicated
```

Comment:
We should try and avoid the term `key` here since we're working with tables and this isn't really a key.